composer.json

```diff
@@ -23,18 +23,18 @@
         "docs": "https://laravel-tenancy.com"
     },
     "require": {
+        "doctrine/dbal": "*",
```

Comment:
Can we revert the alphabetizing of these? It's hard to tell what the actual version change was when the entire line appears changed.

Reply:
Roger.